sportsbook_integration.py

```
@@ -0,0 +1,111 @@
+    def save_lines(self, lines: List[Dict[str, Optional[float]]]):
+        """Save betting lines to SQLite database."""
+        if not lines:
+            logger.warning("No lines to save")
+            return
+        conn = sqlite3.connect(self.db_path)
+        try:
+            for entry in lines:
+                conn.execute(
+                    """
+                    INSERT OR REPLACE INTO betting_lines
+                    (player_id, season, sportsbook, rushing_over_under, receiving_over_under, date_scraped)
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)